### PR TITLE
feat:created single artiscard

### PR DESCRIPTION
**Pull request by GvantsaBarnabishvili**

### CODE REVIEW

File: app/Components/ArtistCard/ArtistCard.module.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  .artistNameFont {
+    font-size: 16px;
+    font-weight: 400;
+  }
+  .albumNameFont {
+    font-size: 16px;
+    font-weight: 400;
+  }
+}
+@media screen and (min-width: 320px) and (max-width: 768px) {
```
Comment (lursmana1) [inline app/Components/ArtistCard/ArtistCard.module.scss:68]: min-width is not needed

File: app/Components/ArtistCard/interfaces/artist-card-props.interface.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { ArtistCardItemsInterface } from './artist-card-items.interfaces';
+
+export interface ArtistCardPropsInterface extends ArtistCardItemsInterface {
+  albumName: string;
+  darkMode?: boolean;
```
Comment (lursmana1) [inline app/Components/ArtistCard/interfaces/artist-card-props.interface.ts:5]: why its optional darkMode is false or true

File: app/Components/ArtisCardsItems/ArtistCardItems.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { ArtistCardItemsInterface } from '../ArtistCard/interfaces/artist-card-items.interfaces';
+
+const ArtistCardItems: ArtistCardItemsInterface[] = [
```
Comment (pailodze) [inline app/Components/ArtisCardsItems/ArtistCardItems.ts:3]: I dont think that artistcardsitems is component oplace it in app/Dummy/artists.ts

File: app/Components/ArtistCard/ArtistCard.module.scss [hunk 4]
```diff
@@ -0,0 +1,92 @@
+@import '../../Colors/common-colors.scss';
+@import '../../Colors/dark-colors.scss';
+@import '../../Colors/light-colors.scss';
+.albumCard {
```
Comment (pailodze) [inline app/Components/ArtistCard/ArtistCard.module.scss:4]: add enter after imports

File: app/Components/ArtistCard/ArtistCard.module.scss [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+  gap: 8px;
+  justify-content: center;
+  padding: 16px;
+  width: 230px;
+  border-radius: 12px;
+  height: 290px;
+  background-color: $background-color-light-mode;
+  color: $primary-black;
+}
+:global(body.dark) .dark {
```
Comment (pailodze) [inline app/Components/ArtistCard/ArtistCard.module.scss:16]: format this file

File: app/Components/ArtistCard/ArtistCard.tsx [hunk 6]
```diff
@@ -0,0 +1,45 @@
+import Image from 'next/image';
+import { TextHtmlTypeEnum } from '../Text/enums/text-html-type.enum';
+import { TextTypeEnum } from '../Text/enums/text-type.enum';
+import styles from './ArtistCard.module.scss';
+import { ArtistCardPropsInterface } from './interfaces/artist-card-props.interface';
+import { ArtistCardType } from './types/artistcard.type';
+import Text from '@/app/Components/Text/Text';
```
Comment (pailodze) [inline app/Components/ArtistCard/ArtistCard.tsx:7]: add enter after imports use prettier

File: app/Components/ArtistCard/ArtistCard.tsx [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        <Text
+          htmlType={TextHtmlTypeEnum.Span}
+          type={TextTypeEnum.SecondaryTextMedium}
+          className={`${styles.albumName} ${styles.albumNameFont}`}
+          color={{
+            lightColor: '#B1B1B1',
+            darkColor: '#B1B1B1',
+          }}
+        >
+          {props.naming}
```
Comment (pailodze) [inline app/Components/ArtistCard/ArtistCard.tsx:38]: props.name

File: app/Components/ArtistCard/interfaces/artist-card-items.interfaces.ts [hunk 8]
```diff
@@ -0,0 +1,6 @@
+export interface ArtistCardItemsInterface {
+  image: string;
+  artistName: string;
+  naming: string;
```
Comment (pailodze) [inline app/Components/ArtistCard/interfaces/artist-card-items.interfaces.ts:4]: name

File: app/Components/ArtistCard/types/artistcard.type.ts [hunk 9]
```diff
@@ -0,0 +1,3 @@
+import { ArtistCardPropsInterface } from '../interfaces/artist-card-props.interface';
```
Comment (pailodze) [inline app/Components/ArtistCard/types/artistcard.type.ts:1]: artist-card.type.ts

File: app/Components/ArtistCards/ArtistCards.module.scss [hunk 10]
```diff
@@ -0,0 +1,13 @@
+.container {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
```
Comment (pailodze) [inline app/Components/ArtistCards/ArtistCards.module.scss:5]: add enter here

File: app/Components/ArtisCardsItems/ArtistCardItems.ts (hunk 3 above)
Comment (GvantsaBarnabishvili) [reply app/Components/ArtisCardsItems/ArtistCardItems.ts:3]: can you please more specific what you mean ?

**Pull request merged**